Core/Utilities/ExecutorDispatcher.py

```diff
@@ -19,125 +22,126 @@ def __init__( self, log = False ):
     self.__execTasks = {}
     self.__taskInExec = {}
 
-  def _internals( self ):
-    return { 'type2id' : dict( self.__typeToId ),
-             'maxTasks' : dict( self.__maxTasks ),
-             'execTasks' : dict( self.__execTasks ),
-             'tasksInExec' : dict( self.__taskInExec ),
-             'locked' : self.__lock.locked() }
+  def _internals(self):
+    return {'type2id': dict(self.__typeToId),
+            'maxTasks': dict(self.__maxTasks),
+            'execTasks': dict(self.__execTasks),
+            'tasksInExec': dict(self.__taskInExec),
+            'locked': self.__lock.locked()}  # pylint: disable=no-member
```

Comment:
That's strange, why dos pylint not finds it ?

Reply:
No idea, really. I tried several combinations and pylint is still wrong. I didn't find a better option.